src/.babelrc

```diff
@@ -1,11 +1,10 @@
 {
   "presets": [
     [
-      "env",
+      "@babel/core",
```

Comment:
Same as above, once you get the `compileJS` task sorted out I don't think you should need this change anymore.